CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 
+## [Unreleased]
+
+### Added
+
+- JSON-RPC: `datastoreusage`: returns the total bytes that are stored under a given key.
+
 
```

Comment:
No, don't edit CHANGELOG.md!

Reply:
Use a Changelog-Added line in your commit message instead.

Reply:
Uh, sorry about that, I added a `Changelog-Added` line to the initial commit message.